pylint/checkers/base_checker.py

```diff
@@ -52,17 +52,25 @@ def __init__(self, linter: PyLinter) -> None:
         _ArgumentsProvider.__init__(self, linter)
 
     def __gt__(self, other: Any) -> bool:
-        """Sorting of checkers."""
+        """Permits sorting checkers for stable doc and tests.
+
+        The main checker is always the first one, then builtin checkers in alphabetical
+        order, then extension checkers in alphabetical order.
+        """
         if not isinstance(other, BaseChecker):
+            # print(f"{other} is not a base checker.")
```

Comment:
Quite a bit of debug statement?

Reply:
I could turn that into comments, but then we'd need to transform it in debug statement again if there's a problem in the future 😄

Reply:
Hmm I think we could remove them here. This feels a bit weird to me

Reply:
Yeah true, comments explaining what the code is doing are bad form. (xor operator that makes it necessary must be bad form too 😅 )